### PR TITLE
zh-CN: Update Simplified Chinese Translation

### PR DESCRIPTION
**Pull request by iGmainC**

### CODE REVIEW

**Review by anlunx — APPROVED** [review]
LGTM. Great translation!

File: po/zh-CN.po [hunk 1]
```diff
@@ -13,7 +13,7 @@ msgstr ""
 
 #: src/SUMMARY.md:3 src/welcome.md:1
 msgid "Welcome to Comprehensive Rust 🦀"
-msgstr ""
+msgstr "欢迎来到全面的 Rust 🦀"
```
[the remark this answers is not in the document]
Comment (mgeisler) [reply po/zh-CN.po:16]: Yes, I suggest adding a translation after the title if you feel it's necessary. The [style guide](https://github.com/google/comprehensive-rust/blob/main/STYLE.md#translations) has a bit of advice about this.

[the remark this answers is not in the document]
Comment (iGmainC) [reply po/zh-CN.po]: I modified it.

**Review by mgeisler — APPROVED** [review]
Thanks for the update!

**Pull request merged**